Source/Krypton Components/Krypton.Navigator/View Layout/ViewLayoutInsetOverlap.cs

```diff
@@ -131,18 +131,24 @@ public override void Layout(ViewLayoutContext context)
             Rectangle childRect = ClientRectangle;
 
             // Find the amount of rounding to apply
+            int tempRounding = Convert.ToInt32(Rounding);
+
             float rounding = Rounding;
 
             // Apply the rounding in the appropriate orientation
             if ((Orientation == VisualOrientation.Top) || (Orientation == VisualOrientation.Bottom))
             {
-                childRect.Width -= Convert.ToInt32(rounding) * 2;
-                childRect.X += Convert.ToInt32(rounding);
+                childRect.Width -= tempRounding * 2;
+                childRect.X += tempRounding;
+
+                rounding = (float)tempRounding;
```

Comment:
So this is effectively an int now..
So What's the purpose of the float ?

Reply:
`RectangleF` already exists ;-)

Reply:
And can be used by `DrawRectangle` etc.

Reply:
So, I can replace ` Rectangle childRect = ClientRectangle;` with ` RectangleF childRect = ClientRectangle;`?

Reply:
Simple assignment:
http://webcache.googleusercontent.com/search?q=cache:8mmcwwjLVdMJ:csharphelper.com/blog/2017/02/convert-a-rectangle-into-a-rectanglef-and-vice-versa-in-c/+&cd=1&hl=en&ct=clnk&gl=uk&client=firefox-b-d

Reply:
`public static implicit operator RectangleF(Rectangle r) => new RectangleF((float) r.X, (float) r.Y, (float) r.Width, (float) r.Height);`